### ISSUE TITLE
Running on unsupported system, point to mysterious "developer_mode" 

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#1829]
*Installation details*
Scylla version (or git commit hash): 1.4.1

Running Scylla 1.4.1 AMI on an t2.micro type results in the following:
```
    t2.micro is not supported instance type!
To continue startup ScyllaDB on this instance, run 'sudo scylla_io_setup' then 'systemctl start scylla-server'.
To run ScyllaDB on supported instance type, run AMI in m3/c3/i2 types.
[centos@ip-172-31-27-37 ~]$ sudo scylla_io_setup
Generating evaluation file sized 10GB... timed out before we could write the entire file. Will continue but accuracy may suffer.
Timed out: timed out before we could write 1GB worth of data. Not enough to continue
/var/lib/scylla did not pass validation tests, it may not be on XFS and/or has limited disk space.
This is a non-supported setup, and performance is expected to be very bad.
For better performance, placing your data on XFS-formatted directories is required.
 To override this error, see the developer_mode configuration option.
```

Last advice is not clear. Where should I look for the "developer_mode configuration option" , and what affect it have?
A better UX will be:
```
To run in dev mode, with inferior performance, use 
sudo /usr/lib/scylla/scylla_dev_mode_setup --developer-mode 1
before starting scylla service

**Comment by tzach** [comment]
With #1702 
```
sudo /usr/lib/scylla/scylla_dev_mode_setup --developer-mode 1
```
can be
```
scylla_dev_mode_setup --developer-mode 1
```

**Comment by tzach** [comment]
Thanks @syuu1228 15b156c solve the issue
If scylla_dev_mode_setup is not in the default path, please open a new issue for it
